barretenberg/cpp/src/barretenberg/translator_vm/translator_prover.cpp

```diff
@@ -74,19 +74,21 @@ void TranslatorProver::commit_to_witness_polynomial(Polynomial& polynomial, cons
  */
 void TranslatorProver::execute_wire_and_sorted_constraints_commitments_round()
 {
-
+    BB_BENCH_NAME("TranslatorProver::execute_wire_and_sorted_constraints_commitments_round");
+    auto batch = key->proving_key->commitment_key.start_batch();
     for (const auto& [wire, label] :
          zip_view(key->proving_key->polynomials.get_wires(), commitment_labels.get_wires())) {
 
-        commit_to_witness_polynomial(wire, label);
+        batch.add_to_batch(wire, label, /*mask for zk?*/ false);
```

Comment:
I'm a little surprised to see no masking here. I guess it must work differently here in some way I dont recall..

Reply:
@maramihali if you feel like weighing in O:)